### PR TITLE
remove unpatched hyperkube images

### PR DESCRIPTION
**Pull request by zhongyi-zhang**

### CODE REVIEW

File: vhdbuilder/packer/install-dependencies.sh [hunk 1]
```diff
@@ -415,28 +415,17 @@ ls -ltr /usr/local/bin >> ${VHD_LOGS_FILEPATH}
 # pull patched hyperkube image for AKS
 # this is used by kube-proxy and need to cover previously supported version for VMAS scale up scenario
 PATCHED_HYPERKUBE_IMAGES="
-1.14.7-hotfix.20200326
 1.14.7-hotfix.20200408.1
-1.14.8-hotfix.20200326
 1.14.8-hotfix.20200529.1
-1.15.7-hotfix.20200326
```
Comment (xuto2) [inline vhdbuilder/packer/install-dependencies.sh:422]: this can be kept, some vmas old cluster still on 1.15.7 may need this.

**Comment by xuto2** [comment]
/lgtm
i will add back v1.15.7-hotfix.20200326

**Comment by zhongyi-zhang** [comment]
Thanks @xuto2.

**Pull request merged**